src/device-manager/java/src/nl/Weave/DataManagement/WdmClientImpl.java

```diff
@@ -179,6 +193,11 @@ private void onRefreshDataComplete()
         requireCompletionHandler().onRefreshDataComplete();
     }
 
+    private void onFetchEventsComplete()
+    {
+        requireCompletionHandler().onFetchEventsComplete(getEvents());
```

Comment:
onFetchEventsComplete should not take getEvents() parameter, the onComplete callback for fetch events just tell app it is successful operation, then app call getEvents to retrieve events

Reply:
The java test program does not work if using onFetchEventsComplete(String events).